README.md

```diff
@@ -1 +1,41 @@
-# js-capstone
\ No newline at end of file
+# mobile-portfolio
+This  project is aimed at building a beautiful porfolio page.
+
+# Built With
+HTML
+CSS
+JAVASCRIPT
+
+# Video presentation
+
+[MOVIES demo](https://youtu.be/PdEa4KJuPNk)
+
+
+
+
+# live Demo
+ To see a live demo kindly visit [MOVIES](https://amklass11.github.io/Mobile-portfolio/)
+
```

Comment:
- [ ] This live demo link does not work at all
![Screenshot from 2022-06-04 09-25-25](https://user-images.githubusercontent.com/56886064/171989356-728b1f2d-56d5-46e7-823c-7feec3eb7005.png)